browse/services/dissemination/__init__.py

```diff
@@ -1,37 +1,22 @@
 """Service to get PDF and other disseminations of an item."""
 from flask import current_app
-from urllib.parse import urlparse
-import google.cloud.storage as storage
 
 from browse.services.documents import get_doc_service
-from arxiv.files.object_store import ObjectStore, GsObjectStore, LocalObjectStore
+from browse.services.global_object_store import get_global_object_store
+from arxiv.files.object_store import ObjectStore
 
 from .article_store import ArticleStore
 
 _article_store: ArticleStore = None  # type: ignore
 # This works because it is thread safe and not bound to the app context.
 
 _object_store: ObjectStore = None  # type: ignore
-# This works because it is thread safe and not bound to the app context.
+
 _genpdf_store: ObjectStore = None  # type: ignore
 
 _latexml_store: ObjectStore = None # type: ignore
 
 
-def get_global_object_store(path: str, global_name: str) -> ObjectStore:
-    """Creates an object store from given path."""
-    store = globals().get(global_name)
-    if store is None:
-        uri = urlparse(path)
-        if uri.scheme == "gs":
-            gs_client = storage.Client()
-            store = GsObjectStore(gs_client.bucket(uri.netloc))
```

Comment:
I didn't notice this before, but I don't love that get_global_object_store has moved to base since it is sort of parameterized on the bucket. 

Some other app might want a obj store for submission source, or log files or audit trails, and those could live in different buckets.

Probably don't change this now but something to keep in mind.